src/core_modules/capture-core/components/WidgetEventSchedule/WidgetEventSchedule.epics.js

```diff
@@ -37,7 +37,7 @@ export const scheduleNewEnrollmentEventEpic = (action$: InputObservable) =>
             }] };
 
             onSaveExternal && onSaveExternal(serverData, uid, onSaveSuccessActionType, onSaveErrorActionType);
```

Comment:
I know you didn't add this here, but we can safely remove the last two arguments here.